tutorial/Execution.md

```diff
@@ -0,0 +1,412 @@
+# A Random Walk Down Executions or: You Could Have Invented Executions or: Learn You An Execution For Greater Good
```

Comment:
I like the title :)

It gets rendered a little strange though, maybe add some new lines?